### DISCUSSION TITLE
Ice 0.10.2

### DISCUSSION THREAD

**Discussion by jordanbaird** [jordanbaird/Ice#235] [Announcements]
## What's New
* Improved the logic for retrieving menu bar items
* Menu bar item images no longer update if the app isn't focused, leading to increased efficiency and the purple screen capture icon appearing less often

## Bug Fixes
* Fixed a crash that would occur when the Ice Bar was longer than the width of the screen
* Fixed a visual glitch where menu bar items wouldn't initially show in the "Menu Bar Items" settings
* Fixed faulty menu bar item image caching
* Fixed a bug that caused the visible section's arranging interface to disappear in "Menu Bar Items" settings when the Ice icon was disabled
* Adjusted the menu bar split appearance shape to have even padding on both sides when using the inset appearance on screens with a notch

**Full Changelog**: https://github.com/jordanbaird/Ice/compare/0.10.1...0.10.2

<hr /><em>This discussion was created from the release <a href='https://github.com/jordanbaird/Ice/releases/tag/0.10.2'>Ice 0.10.2</a>.</em>

**Comment by hohaithuy** [comment]
Awesome! I'm super excited for the group items feature to roll out soon. The screen is a bit crowded with all the visible items right now, so grouping them will totally tidy things up and make everything look super sleek!

**Comment by userandrewdrew** [comment]
When playing games and right clicking where the meny bar section would be in full screen the mouse glitches, I think it is because of the "Edit Menu Bar Appearance" setting opening. Is it possible to turn this setting off or perhaps fix this somehow? :D
